loopchain/peer/peer_service.py

```diff
@@ -348,16 +308,3 @@ def close(self):
 
     def _load_channel_infos(self):
         self._channel_infos = self._get_channel_infos()
-
-    async def change_node_type(self, node_type):
```

Comment:
The PeerService no longer manages the node_type, so it doesn't matter whether the node_type has changed or not. When node_type is changed, it is managed by the corresponding channel because it is related only to a specific channel.